csharp/src/Google.Protobuf/Compatibility/TypeExtensions.cs

```diff
@@ -59,7 +60,15 @@ internal static bool IsAssignableFrom(this Type target, Type c)
         /// including inherited properties or null if there is no such public property.
         /// Here, "public property" means a property where either the getter, or the setter, or both, is public.
         /// </summary>
-        internal static PropertyInfo GetProperty(this Type target, string name)
+#if NET5_0_OR_GREATER
```

Comment:
Is this whole file only needed for targeting netstandard1.1? Would it be better to change the ifdef around this whole file to `#if NETSTANDARD1_1` instead of these changes?

Reply:
This assembly is using TypeInfo everywhere, rather than only on netstandard1.1. I don't want to mess with that in this PR.